incubator/binding-openapi-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/asyncapi/internal/config/OpenapiAsyncCompositeBindingAdapter.java

```diff
@@ -49,8 +51,10 @@
 public final class OpenapiAsyncCompositeBindingAdapter implements CompositeBindingAdapterSpi
 {
     private static final Pattern JSON_CONTENT_TYPE = Pattern.compile("^application/(?:.+\\+)?json$");
+    private static final Pattern PARAMETER_PATTERN = Pattern.compile("\\{([^}]*)\\}");
```

Comment:
```suggestion
    private static final Pattern PARAMETER_PATTERN = Pattern.compile("\\{([^}]+)\\}");
```
We require at least one character for parameter name, agree?